pennylane/labs/intermediate_reps/rowcol.py

```diff
@@ -0,0 +1,658 @@
+               the Steiner tree :math:`T` starting from :math:`i` as root. If ``P[c, i]=1`` for node
+               :math:`c` in the traversal and ``P[p, i]=0`` for parent node :math:`p` of node :math:`c`,
+               add row ``P[c]`` to row ``P[p]`` and append ``CNOT([c, p])`` to ``L``.
+            c. Post-order traverse :math:`T` again starting from :math:`i`. For each node :math:`p`
+               in the traversal, add row ``P[p]`` to row ``P[c]``, where :math:`c` is the child node
+               of :math:`p`, and append ``CNOT([p, c])`` to ``L``.
+
+        #. Eliminate row ``P[i]``
+
+            a. Construct :math:`S'` such that the corresponding rows of :math:`P` add up to the
+               :math:`i`\ th basis vector, :math:`\sum_{j\in S'} P[j] = e_i`. Find a Steiner tree
+               :math:`T' \supseteq S'`.
+            b. `Pre-order traverse <https://en.wikipedia.org/wiki/Tree_traversal#Pre-order,_NLR>`__
+               the Steiner tree :math:`T'` starting from :math:`i` and for each node :math:`c` in
+               the traversal for which :math:`c \notin S'`, add row :math:`P[c]` to row :math:`P[p]`,
+               where :math:`p` is the parent node of :math:`c`, and append ``CNOT([c, p])`` to ``L``.
+            c. Post-order traverse :math:`T` starting from :math:`i`. For each node :math:`c`
+               in the traversal, add row ``P[c]`` to row ``P[p]``, where :math:`p` is the parent node
+               of :math:`c`, and append ``CNOT([c, p])`` to ``L``.
+
+        #. Delete vertex :math:`i` from :math:`G` and ignore row and column with index :math:`i` in
+           the following. Go to step 1.
+        #. Revert the list ``L`` of CNOT gates in order to go from the circuit that transforms
+           :math:`P` to the identity to the circuit that implements :math:`P` starting from the
+           identity.
+
+    .. details::
+        :title: Manual example
+        :href: manual-example
+
+        We walk through example 1 in arXiv:1910.14478 by Wu et al., which was demonstrated
+        in code above, in detail. The steps are numbered according to the algorithm overview
+        above.
+        We restrict ourselves to the following connectivity graph.
+
+        .. code-block:: python
+
+            (0) - (3) - (4)
+                   |
+                  (2)
+                   |
+                  (1)
+
+        We look at a CNOT circuit with the following parity matrix. The first column is
+        highlighted as we are going to start by eliminating it (``(0)`` is not a cut vertex).
+
+        .. math::
+
+            P = \begin{pmatrix}
+                \color{blue}1 & 1 & 0 & 1 & 1 \\
+                \color{blue}0 & 0 & 1 & 1 & 0 \\
+                \color{blue}1 & 0 & 1 & 0 & 1 \\
+                \color{blue}1 & 1 & 0 & 1 & 0 \\
+                \color{blue}1 & 1 & 1 & 1 & 0
+            \end{pmatrix}
+
+        *Step 1: Pick vertex ``i=0``*
```

Comment:
This part is not rendering correctly. Maybe * and `` are not compatible together

Reply:
Good catch, thanks! I now made just the _Step1:_ italic, hopefully that will look okay :)